jujugui/static/gui/src/app/app.js

```diff
@@ -1480,6 +1483,45 @@ YUI.add('juju-gui', function(Y) {
     },
 
     /**
+      Creates new API client instances for Romulus services.
+      Assign them to the "plans" and "terms" app properties.
+      This function is idempotent.
+
+      @method _setupRomulusServices
+      @param {Object} config The GUI application configuration.
+      @param {Object} jujulib The Juju API client library.
+      @param {Object} storage The place where to store macaroons.
+    */
+    _setupRomulusServices: function(config, jujulib, storage) {
+      if (!config) {
+        // We are probably running tests.
+        return;
+      }
+      // If one services is already set up, then assume the other is as well.
```

Comment:
s/services/service

Why would one be set up already?

I think it's also going to be dangerous to assume that both are created just because one exists.